backend/workflow_manager/workflow/views.py

```diff
@@ -207,20 +196,8 @@ def execute(
                 make_execution_response(execution_response),
                 status=status.HTTP_200_OK,
             )
-        except (InvalidRequest, WorkflowExecutionError) as exception:
```

Comment:
I think we need these two excpetion handler,  For workflow exception not for pipeline

Reply:
Anyway these will be caught and re-raised in the generic exception block. There's no need to log the error and re-raise the same, it'll be done by the middleware anyway